data/recommendation/style.scss

```diff
@@ -34,18 +34,16 @@ body {
     width: 100%;
   }
   .image-box {
-    width: 10%;
+    align-items: center;
+    display: flex;
```

Comment:
`.image-box` only has one child, right? The image? Using flexbox with only one image seems strange to me. Flexbox's main benefits come from how it deals with layout of multiple items in relation to each other. I'm not sure this is the right way to fix this.